File: src/jp/mcbe/fuyutsuki/Texter/text/Sendable.php
<?php

declare(strict_types=1);

namespace jp\mcbe\fuyutsuki\Texter\text;

use pocketmine\player\Player;
use pocketmine\world\World;

interface Sendable {

	public function sendToPlayer(Player $player, SendType $type): void;

	public function sendToPlayers(array $players, SendType $type): void;

	public function sendToWorld(World $world, SendType $type): void;

}

File: src/main/java/jp/mcbe/fuyutsuki/Texter/text/Sendable.java
package jp.mcbe.fuyutsuki.Texter.text;

import org.powernukkitx.Player;
import org.powernukkitx.level.Level;

import java.util.Collection;

public interface Sendable {

	void sendToPlayer(Player player, SendType type);

	void sendToPlayers(Collection<? extends Player> players, SendType type);

	void sendToLevel(Level level, SendType type);
}